### PR TITLE
Add talks and videos from the last Prague meetup

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: series/praha-pyvo/events/2019-09-18.yaml [hunk 1]
```diff
@@ -9,24 +9,53 @@ venue: na-venecku
 urls: []
 talks:
 
-- title: Exceptions in Python
+- title: Výjimky
   speakers:
-  - ??
+  - Petr Viktorin
+  coverage:
+  - video: https://www.youtube.com/watch?v=gRaL4VwFph4
```
Comment (hroncok) [inline series/praha-pyvo/events/2019-09-18.yaml]: I remember you've said: Don't record the slides, record me, I'm recording my screen.

Yet the video doesn't have your screen. What happened?

Comment (encukou) [reply series/praha-pyvo/events/2019-09-18.yaml]: The recording failed. In the beginning projector was showing weird colors, so we connected things up in a non-standard way, and the screen recording came out solid blue.
Thanks for not following my request; you saved the day!

Comment (hroncok) [reply series/praha-pyvo/events/2019-09-18.yaml]: I actually tried to follow it. So I guess my poor recording skills were actually useful :+1:

File: series/praha-pyvo/events/2019-09-18.yaml [hunk 2]
```diff
@@ -9,24 +9,53 @@ venue: na-venecku
...
+  coverage:
+  - slides: https://docs.google.com/presentation/d/1hxiHR9RS8mvRR0A9mlraiWWfFupMOhAHDehqK4JH9lA/edit?usp=sharing
+  - video: https://www.youtube.com/watch?v=4qeKRVJDDZo
 
-- title: Lightning talky
-  description: |
-    Kromě grilovačky budou lightning talky. Můžeš jeden mít i ty,
-    viz [pyvo.cz#info-for-speakers](https://pyvo.cz/#info-for-speakers).
-    Registrace nutná na [Lightning Talk – bit.ly/prpylight](https://bit.ly/prpylight).
+- title: výuka Pythonu pre začiatočníkov & OWASP
```
Comment (hroncok) [inline series/praha-pyvo/events/2019-09-18.yaml]:
```suggestion
- title: Výuka Pythonu pre začiatočníkov & OWASP
```

**Pull request merged**